tests/stockfish/test_models.py

```diff
@@ -408,3 +405,43 @@ def test_make_moves_transposition_table_speed(self):
             total_time_calculating_second += default_timer() - start
 
         assert total_time_calculating_first < total_time_calculating_second
+
+    def test_benchmark_result_with_defaults(self, stockfish):
+        params = stockfish.BenchmarkParameters()
+        result = stockfish.benchmark(params)
+        # result should contain the last line of a successful method call
+        assert result.split(" ")[0] == "Nodes/second"
+
+    def test_benchmark_result_with_valid_options(self, stockfish):
+        params = stockfish.BenchmarkParameters(
+            ttSize=64, threads=2, limit=1000, limitType="movetime", evalType="classical"
+        )
+        result = stockfish.benchmark(params)
+        # result should contain the last line of a successful method call
+        assert result.split(" ")[0] == "Nodes/second"
+
+    def test_benchmark_result_with_invalid_options(self, stockfish):
+        params = stockfish.BenchmarkParameters(
+            ttSize=2049,
+            threads=0,
+            limit=0,
+            fenFile="./fakefile.fen",
+            limitType="fghthtr",
+            evalType="",
+        )
+        result = stockfish.benchmark(params)
+        # result should contain the last line of a successful method call
+        assert result.split(" ")[0] == "Nodes/second"
+
+    def test_benchmark_result_with_invalid_type(self, stockfish):
+        params = {
```

Comment:
```suggestion
        params = stockfish.BenchmarkParameters(
```

Reply:
This test is to make sure the bechmark() call will still work even if the user trys to pass in arguments that are not of BenchmarkParameters type. that is why it is a normal dictionary.

if you think this test case is not needed, we can remove it instead of making the params the benchmarkParameters type.

Reply:
Now I understand your idea, thanks :)